src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/ProjectPropertiesExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
+{
+    internal static class ProjectPropertiesExtensions
+    {
+        /// <summary>
+        /// Saves the unevaluated value of <paramref name="propertyName"/>, if any, to <paramref name="storage"/>.
+        /// </summary>
+        public static async Task RememberValueIfCurrentlySet(this IProjectProperties properties, string propertyName, ITemporaryPropertyStorage storage)
```

Comment:
>RememberValueIfCurrentlySet [](start = 33, length = 27)

I think SaveValueIfCurrentlySet would be more descriptive about what's going on when reading the code, but that might just be me

Reply:
👍🏻